<launch>

    <node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>

    <node name="wheel_visual_servo" pkg="visual_servo_control" type="III_visual_servo_with_wheel" output="screen">
        <param name="Kp" type="double" value="5.0"/>
        <param name="Kd" type="double" value="0.0"/>
        <param name="Kf_r0" type="double" value="0.5"/>
        <param name="Kf_q0" type="double" value="0.1"/>
        <param name="publisher_topic" type="string" value="/cmd_vel"/>
        <param name="kalman_input_topic" type="string" value="/visual_servo/kalman_input"/>
        <param name="kalman_output_topic" type="string" value="/visual_servo/kalman_output"/>
        <param name="omega_raw_topic" type="string" value="/visual_servo/raw_omega_cam"/>
        <param name="omega_input_topic" type="string" value="/can_receive_1/end_effector_omega"/>
        <param name="cfg_file_name" type="string" value="/home/nvidia/ws/src/6_controller/cfg/camera_tracking_camera_calib.yaml"/>
    </node>
</launch>
